p2p/pkg/store/store.go

```diff
@@ -308,3 +318,41 @@ func (s *Store) Len() int {
 
 	return s.Tree.Len()
 }
+
+type TxnDetails struct {
+	Hash    common.Hash
+	Nonce   uint64
+	Created int64
+}
+
+func (s *Store) Save(ctx context.Context, txHash common.Hash, nonce uint64) error {
```

Comment:
```suggestion
func (s *Store) SaveTxnDetails(ctx context.Context, txHash common.Hash, nonce uint64) error {
```

Reply:
Thank you for your comments around Save and Update. I understand the reasoning behind the naming related to the txmonitor.Saver interface. However, I still believe that renaming the function to SaveTxnDetails would improve clarity.

While the current name might seem logical within the txmonitor package, it causes some confusion within store.go. The lack of logical separation between methods in store.go makes it hard to understand the purpose of the Save method without referring back to the txmonitor package. Renaming it to SaveTxnDetails would make the function's intent clearer within the context of store.go.

Reply:
Thank you for the detailed comment! It makes sense to me now that the store wasn't supposed to implement this interface.